go.mod

```diff
@@ -120,6 +120,7 @@ require (
 	gopkg.in/httprequest.v1 v1.2.1
 	gopkg.in/ini.v1 v1.10.1
 	gopkg.in/juju/blobstore.v2 v2.0.0-20160125023703-51fa6e26128d
+	gopkg.in/juju/charm.v6 v6.0.0-20190729113111-40ffcf7d10e5
```

Comment:
This is the wrong charm. Probably want to do a `go mod tidy`

Reply:
doh, thanks